blog_post_collaboration/spark_streaming_dataset.md

```diff
@@ -0,0 +1,367 @@
+# A new Kedro dataset for Spark Structured Streaming
+
+This article describes a new [Kedro dataset for Spark Structured Streaming](https://github.com/kedro-org/kedro-plugins/tree/main/kedro-datasets/kedro_datasets/spark). [Spark Structured Streaming](https://spark.apache.org/docs/latest/structured-streaming-programming-guide.html) is built on the Spark SQL engine. You can express your streaming computation the same way you would express a batch computation on static data, and the Spark SQL engine will run it incrementally and continuously and update the final result as streaming data continues to arrive. 
+
+## Introduction
+
+This needs to be written:
+* What this post solves for the reader, and who would be interested in learning about this.
+* Limitations of the dataset (if any). Proof of concept only??
+* Link to source code and further information about Kedro datasets.
+* What the following sections show: set up a project, configure the data catalog
```

Comment:
This post is to give data practitioners a guidance on how to use streaming datasets, 
Limitations:
1. Not ready for service broker e.g., Kafka, as the additional Jar package is required
2. Schema is enforced